Gemfile

```diff
@@ -52,6 +55,8 @@ group :development do
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
   gem 'figaro'
+  gem 'factory_bot_rails'
```

Comment:
You can move the gems up to the group :development, :test part and then you don't have to have them twice